### PR TITLE
END FUND: business status of the wards, on the map view and on the legend 

### PR DESCRIPTION
**Pull request by haseemVD**
This PR addresses part of #1567 (only sidebar)

Screenshot 2021-04-30 at 2 38 56 PM

Screenshot 2021-05-04 at 2 16 54 PM

### CODE REVIEW

**Review by ciremusyoka — CHANGES_REQUESTED** [review]
Should the units for the progress bars be `structure(s)`?
e.g `Progress: 6989 of 8000 structures(s) (87%)`

Also should the progress values for the last three items have units?

**Comment by haseemVD** [comment]
> progress

@samkanga Can you please confirm.

File: src/configs/env.ts [hunk 1]
```diff
@@ -199,6 +199,14 @@ export const SUPERSET_IRS_LITE_REPORTING_INDICATOR_ROWS =
...
 export type SUPERSET_IRS_LITE_REPORTING_INDICATOR_ROWS = typeof SUPERSET_IRS_LITE_REPORTING_INDICATOR_ROWS;
 
+export const SUPERSET_MDA_LITE_REPORTING_INDICATOR_ROWS =
+  process.env.REACT_APP_SUPERSET_MDA_LITE_REPORTING_INDICATOR_ROWS || 'zambia2020';
+export type SUPERSET_MDA_LITE_REPORTING_INDICATOR_ROWS = typeof SUPERSET_MDA_LITE_REPORTING_INDICATOR_ROWS;
+
+export const SUPERSET_MDA_LITE_REPORTING_INDICATOR_STOPS =
+  process.env.REACT_APP_SUPERSET_MDA_LITE_REPORTING_INDICATOR_STOPS || 'zambia2020';
+export type SUPERSET_MDA_LITE_REPORTING_INDICATOR_STOPS = typeof SUPERSET_MDA_LITE_REPORTING_INDICATOR_STOPS;
+
```
Comment (ciremusyoka) [inline src/configs/env.ts:209]: Should the default value here be `zambia2020`?
Note: MDA-Lite plans are for kenya.

Comment (haseemVD) [reply src/configs/env.ts:209]: @ciremusyoka 
followed IRS lite implementation, Let me know if this needs to be changed for MDA lite.

File: src/containers/pages/MDALite/map/helpers.ts [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+  ['Completed', TASK_GREEN],
+  ['Visited but not completed', TASK_ORANGE],
+  ['Not Visited', TASK_YELLOW],
+];
+
+/** IRS Lite Indicator stops
+ * These are all the indicator stops for IRS Lite that we know about.
+ */
+export const MDALiteIndicatorStops: { [key: string]: string[][] } = {
+  zambia2020: defaultIndicatorStop,
```
Comment (ciremusyoka) [inline src/containers/pages/MDALite/map/helpers.ts:15]: This should not be `zambia2020`.

File: src/containers/pages/MDALite/map/tests/fixtures.ts [hunk 3]
```diff
@@ -0,0 +1,66 @@
+export const subcountyData = [
```
Comment (ciremusyoka) [inline src/containers/pages/MDALite/map/tests/fixtures.ts:1]: Is it possible to use the fixtures used for the other MDA-Lite tests?

File: src/configs/env.ts (hunk 1 above)
Comment (ciremusyoka) [reply src/configs/env.ts:209]: The reason why `zambia2020` was used is because IRS was used for Zambia.

File: src/configs/env.ts [hunk 4]
```diff
@@ -199,6 +199,14 @@ export const SUPERSET_IRS_LITE_REPORTING_INDICATOR_ROWS =
   process.env.REACT_APP_SUPERSET_IRS_LITE_REPORTING_INDICATOR_ROWS || 'zambia2020';
```
Comment (ciremusyoka) [inline src/configs/env.ts:199]: New envs are documented [here](https://github.com/onaio/reveal-frontend/blob/master/docs/env.md) and also added to the [.env.sample](https://github.com/onaio/reveal-frontend/blob/master/.env.sample) file.

**Comment by samkanga** [comment]
@haseemVD the units for coverage should be people `e.g. 6989 of 8000 People`
Total treated  - units should be `People`
Drugs Administered - units should be `Tablets`
Drugs Damaged - units should be `Tablets`

**Comment by haseemVD** [comment]
Updated units:

Screenshot 2021-05-03 at 11 30 31 AM

File: src/containers/pages/MDALite/map/helpers.ts [hunk 5]
```diff
@@ -0,0 +1,95 @@
...
+}
+
+/** the indicator row type */
+export type IndicatorRows = MDAIndicatorRowItem[];
+
+/** IRS Indicator rows
+ * These are all the indicator rows for IRS that we know about.
+ */
+export const MDAIndicatorRows: { [key: string]: IndicatorRows } = {
+  kenya2021: [
```
Comment (ciremusyoka) [inline src/containers/pages/MDALite/map/helpers.ts:39]: Let's move the titles here to the lang file.

**Comment by haseemVD** [comment]
Screenshot 2021-05-04 at 2 16 54 PM

@samkanga This is the final version.

**Comment by samkanga** [comment]
@haseemVD @ciremusyoka the final piece looks great!

File: src/configs/strings/en.json [hunk 6]
```diff
@@ -519,6 +519,14 @@
     "message": "Intervention Type",
     "description": "form label: for the type of intervention plan"
   },
+  "PEOPLE": {
+    "message": "people",
+    "description": "lable: mda lite indicator unit"
```
Comment (ciremusyoka) [inline src/configs/strings/en.json:524]: Typo on label.

Comment (ciremusyoka) [inline src/configs/strings/en.json:524]: Typo on label.

File: .env.sample [hunk 7]
```diff
@@ -194,3 +194,6 @@ REACT_APP_HIDE_PLAN_FORM_FIELDS_ON_EDIT=''
 
 REACT_APP_AUTO_SELECT_FI_CLASSIFICATION=true
 REACT_APP_PLAN_LIST_SHOW_FI_REASON_COLUMN=true
+
+REACT_APP_SUPERSET_MDA_LITE_REPORTING_INDICATOR_ROWS=kenya2021
+REACT_APP_SUPERSET_MDA_LITE_REPORTING_INDICATOR_STOPS=kenya2021
```
Comment (p-netm) [inline .env.sample:199]: got a missing newLine here.

File: src/containers/pages/MDALite/map/helpers.ts [hunk 8]
```diff
@@ -0,0 +1,103 @@
...
+    {
+      accessor: 'damaged',
+      listDisplay: true,
+      title: DRUGS_DAMAGED,
+      unit: TABLET,
+    },
+  ],
+};
+
+export const getMDAIndicatorRows = (indicatorRows: any, subcountyData: any) => {
```
Comment (p-netm) [inline src/containers/pages/MDALite/map/helpers.ts:85]: @haseemVD was any here intentional?

File: src/containers/pages/MDALite/map/index.tsx [hunk 9]
```diff
@@ -260,6 +302,57 @@ const MDALiteMapReport = (props: MDALiteMapProps & RouteComponentProps<RoutePara
             )}
           </div>
         </Col>
+        <Col xs={3}>
+          <div className="mapSidebar">
```
Comment (p-netm) [inline src/containers/pages/MDALite/map/index.tsx:306]: Should we add some tests to gaurd for regressions for fields in this column

**Review by p-netm — COMMENTED** [review]
LGTM, a few comments though.

**Review by ciremusyoka — APPROVED** [review]

**Review by moshthepitt — APPROVED** [review]

**Pull request merged**